pkg/globalconfig/schema.json

```diff
@@ -2,28 +2,9 @@
   "$id": "https://json.schemastore.org/base.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "DDEV global config",
-  "description": "Schema for DDEV global config for version 1.22",
+  "description": "Schema for DDEV global config for version 1.23",
```

Comment:
I wonder if it's useful adding the version. Just creates maintenance problems.

Reply:
Agreed, I removed the version.